src/fontra/views/editor/visualization-layer-definitions.js

```diff
@@ -172,6 +177,29 @@ registerVisualizationLayerDefinition({
   },
 });
 
+registerVisualizationLayerDefinition({
+  identifier: "fontra.glyph.locking",
+  name: "Glyph locking",
+  selectionMode: "editing",
+  userSwitchable: true,
+  zIndex: 700,
+  screenParameters: {
+    iconSize: 15,
+  },
+  colors: { strokeColor: "#000C" },
+  colorsDarkMode: { strokeColor: "#FFFC" },
+  draw: (context, positionedGlyph, parameters, model, controller) => {
+    if (!!positionedGlyph.varGlyph.glyph.customData["fontra.glyph.locked"]) {
+      const box = positionedGlyph.unpositionedBounds;
```

Comment:
Please use `positionedGlyph.glyph.controlBounds` instead, and check whether it is null/undefined, fall back to the baseline if so.

(The unpositioned bounds do some weird things for empty glyphs, which is confusing here.)

Reply:
done